addons/stock/models/stock_move.py

```diff
@@ -1730,3 +1731,26 @@ def _adjust_procure_method(self):
     def _show_details_in_draft(self):
         self.ensure_one()
         return self.state != 'draft' or (self.picking_id.immediate_transfer and self.state == 'draft')
+
+    def _trigger_scheduler(self):
+        """ Check for auto-triggered orderpoints and trigger them. """
+        orderpoints_by_company = defaultdict(lambda: self.env['stock.warehouse.orderpoint'])
+        for move in self:
+            orderpoint = self.env['stock.warehouse.orderpoint'].search([
+                ('product_id', '=', move.product_id.id),
+                ('trigger', '=', 'auto'),
+                ('location_id', 'parent_of', move.location_id.id),
+            ], limit=1)
```

Comment:
I know it was already the case before but I would add 'company_id in search and use it into the defaultdict

Reply:
@Whenrow Isn't location_id enough for that?

Reply:
@yhu-odoo In most cases location is enough but company_id is optional on it so you could theoretically share a location between two orderpoints with a different company.

Reply:
Makes sense + done